recipes-bsp/acpi-tables/samples/edison/leds.asl

```diff
@@ -0,0 +1,88 @@
+/*
+ * Intel Edison
+ *
+ * This table is modified from the Minnowboard Turbot leds.asl for Edison.
```

Comment:
Here Turbot is mentioned but in the commit message you put MAX. Which one is correct?

Reply:
Good question. I took from samples/minnowboard-max/leds.asl which in the first line has:
* http://wiki.minnowboard.org/MinnowBoard_Turbot

To be honest I don't know the difference between Minnowboard/Max/Turbot. But I guess this should really be Max?

Reply:
Minnoboard = v1
... MAX = v2
... Turbot = v3

AFAIU this.

Okay, just check if turbot has that led and use it, or instead you may refer to the file name relative to the source root.